### ISSUE TITLE
[BUG] A seemingly critical bug in evaluate_on_environment()

### ISSUE THREAD

**Issue by tominku** [takuseno/d3rlpy#189]
In the current version, it does not normalize the observation when evaluating the agent in the environment. I think it should normalize the observation if the algo was trained with normalized observations via scaler.

**current version**
evaluate_on_environment(...):
...
    action = algo.predict([observation])[0]
...

**needs to be modified as follows**
evaluate_on_environment(...):
...
if scaler is not None:
          observation = (
              scaler.transform(
                  torch.tensor(observation, device=device)
              )
              .cpu()
              .numpy()[0]
          )
      action = algo.predict([observation])[0]
...

**Comment by takuseno** [comment]
it is normalized inside algo.
https://github.com/takuseno/d3rlpy/blob/1bdbaed1d3a3bb1d6aae6e60f16b64b16c8ffdb3/d3rlpy/algos/torch/base.py#L53

**Comment by takuseno** [comment]
Let me close this issue since the issue does not exist.

**Comment by tominku** [comment]
Thanks. I was not able to notice that there exists normalization code for test time. It was a bit tough to find the code snippet since the overall codes are a little bit spread over different classes. It is a bit hard for me to navigate many classes to find a code snippet I want to look at. But thank you anways for this good code base.
